installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -109,5 +109,6 @@
         "ENABLE_SLACK": "false",
         "SLACK_USER": "{slack_user}",
         "SLACK_TOKEN": "{slack_token}"
-    }
+    },
+    "AWS_TAGS": []
```

Comment:
Move the key under the node $.AWS and perhaps just call it TAGS so basically it becomes $.AWS.TAGS.